meadow/client/static/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE HTML>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+
+        <title>Plutus Playground</title>
```

Comment:
Probably should say Meadow for the time being.

Reply:
will change in my latest PR